R/derive_var_shift.R

```diff
@@ -0,0 +1,109 @@
+#' Derive Shift
+#'
+#' Derives a character shift variable containing concatenated shift in
+#' values based on user-defined pairing, e.g., shift from baseline to
+#' analysis value, shift from baseline grade to analysis grade, ...
+#'
+#' @param dataset Input dataset
+#'
+#'   The columns specified by `from_var` and the `to_var`parameters are expected.
+#'
+#' @param new_var Name of the character shift variable to create.
+#'
+#' @param from_var Variable containing value to shift from.
+#'
+#' @param to_var Variable containing value to shift to.
+#'
+#' @param post_baseline_condition Condition to identify post-baseline records. If not
+#' specified, `new_var` is populated for all records.
```

Comment:
I think we usually use `filter` rather then `condition`. Might be worth checking and to be consistent throughout the package.

Reply:
would agree to use '`filter` rather than `condition`